src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncIteratorTests.cs

```diff
@@ -4140,6 +4144,105 @@ async Unknown local()
                 );
         }
 
+        [ConditionalFact(typeof(WindowsDesktopOnly))]
```

Comment:
What bug is tracking this being enabled on CoreClr again?

Reply:
[https://github.com/dotnet/roslyn/issues/30438](https://github.com/dotnet/roslyn/issues/30438)

---
In reply to: [244785283](https://github.com/dotnet/roslyn/pull/31764#discussion_r244785283) [](ancestors = 244785283)